### PR TITLE
testing pr - start of model encryption

### PR DESCRIPTION
**Pull request by JoshuaMitchell**

### CODE REVIEW

File: app/models/registrant.py [hunk 1]
```diff
@@ -14,7 +26,40 @@ class Registrant(db.Model):
...
+    lang = db.Column(db.String()) #enum? (values?)
+    signed_at = db.Column(db.DateTime, default=datetime.utcnow()) #converted to local time on image generated submission
+
     registration = db.Column(db.String())
+    #create key from environmental key
+    #json stringyify dictionary and encrypt
+
+    @hybrid_property
+    def registration_value(self):
+        return encryptem(self.registration)
```
[the remark this answers is not in the document]
Comment (JoshuaMitchell) [reply app/models/registrant.py]: Yes.  I wasn't actually using it and didn't catch it in my tests either. Thanks!

File: app/models/tests/test_registrant.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        county="Johnson",
+        registration_value = data,
+    )
+    session.add(new_registrant)
+    session.commit()
+    registrant = session.query(Registrant).first()
+    assert isinstance(registrant.registration, (dict)) == False
+    assert registrant.lang == 'en'
+
+    decrypted = json.loads(decryptem(registrant.registration))
```
[the remark this answers is not in the document]
Comment (JoshuaMitchell) [reply app/models/tests/test_registrant.py]: I've updated the registration_value to return a python dictionary of the decrypted data.

File: app/models/tests/test_registrant.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    session.commit()
+
+
+    registrant = session.query(Registrant).first()
+    #confirm that registration was modified from dictionary value
+    assert isinstance(registrant.registration, (dict)) == False
+
+    #confirm that registration_value is returned decrypted and marshalled to dictionary
+    assert isinstance(registrant.registration_value, (dict)) == True
+    assert registrant.registration_value.get('name_first') == "foo"
```
Comment (karpet) [inline app/models/tests/test_registrant.py]: lgtm!

File: app/models/tests/test_registrant.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    
+    #confirm that registration was encrypted and can be decrypted
+    decrypted = json.loads(decryptem(registrant.registration))
```
Comment (karpet) [inline app/models/tests/test_registrant.py]: Do you still need this now that you've changed the test on line 50?

**Comment by karpet** [comment]
@JoshuaMitchell this is looking good. Just a question about whether a particular test is still needed, and then I think we're good to merge.

File: app/models/tests/test_registrant.py (hunk 4 above)
Comment (JoshuaMitchell) [reply app/models/tests/test_registrant.py]: Yes I think so.  That is verifying that the registrant.registration (what is actually stored the db) is in fact encrypted bytes.

**Pull request merged**